src/index.ts

```diff
@@ -211,7 +212,7 @@ export default class OtaClient {
         }
         const timestamp = await this.getManifestTimestamp();
         url += `?timestamp=${timestamp}`;
-        return this.httpClient.get(url);
+        return this.httpClient.get<string | Record<string, any>>(url).catch(() => null);
```

Comment:
It still wont cover case for binary data and if json is an array of objects (not sure if this is actually a real life scenario).  So we will need to add `any` here

Reply:
Alright I guess that can be something for a future PR, I'll revert and only keep null